coap/examples/coap_client/main/coap_client_example_main.c

```diff
@@ -371,6 +374,9 @@ static void coap_example_client(void *p)
     unsigned char token[8];
     size_t tokenlength;
 
+    /* Initialize libcoap library */
+    coap_startup();
```

Comment:
Is this a breaking change? I mean example that links against newer component version would not have this call as part of their initialization sequence. Any other place where this could be placed, say in port layer?

Reply:
`coap_startup()` has been around for a very long time and should have been in the examples from day 1.  It is called from `coap_new_context()`, but users could call other `coap_*()` functions before `coap_new_context()` which may give non-initialization issues.

Reply:
Okay, in that case we are good here.